ci/gpu/build.sh

```diff
@@ -128,7 +128,9 @@ GTEST_OUTPUT="xml:${WORKSPACE}/test-results/libcuml_cpp/" ./test/ml
 logger "Python pytest for cuml..."
 cd $WORKSPACE/python
 
-pytest --cache-clear --junitxml=${WORKSPACE}/junit-cuml.xml -v -s -m "not memleak" --durations=50 --timeout=300
+pytest --cache-clear --junitxml=${WORKSPACE}/junit-cuml.xml -v -s -m "not memleak" --durations=50 --timeout=300 --ignore=cuml/test/dask
+
+timeout 7200 sh -c "pytest cuml/test/dask --cache-clear --junitxml=${WORKSPACE}/junit-cuml-mg.xml -v -s -m 'not memleak' --durations=50 --timeout=300"
```

Comment:
Does this cause issues on the jenkins side that we now have two different .xml files?
Maybe keep a single command but do a hard timeout at like 3 hrs?

Reply:
@JohnZed, it shouldn't matter. Jenkins will aggregate all the test results that match the file patterns shown below. so as long as the results are published to the same location (which it looks like they would be), there shouldn't be any issues here.

![image](https://user-images.githubusercontent.com/7400326/81201584-6ffc6d80-8f93-11ea-8e92-da80c08682c7.png)

Reply:
actually, we'd need to make sure they don't overwrite each other. so we'd have to make sure they have different file names at least.